src/Mono.Android/Xamarin.Android.Net/AndroidClientHandler.cs

```diff
@@ -272,10 +277,10 @@ string EncodeUrl (Uri url)
 				if(Logger.LogNet)
 					Logger.Log (LogLevel.Info, LOG_APP, " cancelled");
 
-				httpConnection.Disconnect ();
+				await DisconnectAsync (httpConnection);
```

Comment:
By not calling `ConfigureAwait(false)` on the Task returned by DisconnectAsync, don't you negate the value of that method's `ConfigureAwait` call?